src/test/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatterSpec.scala

```diff
@@ -166,9 +171,7 @@ class TSVFormatterSpec extends FreeSpec with ScalaFutures with Matchers with Ins
 
   }
 
-  private def testEntityDataSet(entityType: String, entities: List[RawlsEntity], requestedHeaders: Option[IndexedSeq[String]]) = {
-    val headerRenamingMap: Map[String, String] = ModelSchema.getAttributeExportRenamingMap(entityType)
-      .getOrElse(Map.empty[String, String])
+  private def testEntityDataSet(entityType: String, entities: List[RawlsEntity], requestedHeaders: Option[IndexedSeq[String]], expectUpdateTsv: Boolean = false) = {
```

Comment:
I'd rather the last parameter be `expectedTsvType: TsvType = TsvType.ENTITY` than a `Boolean`. That would clarify the intent where this is called and also reduce the if/else below to:
```
    headers(0) should be(s"${expectedTsvType}:${entityType}_id")
```